frontend/src/pages/index.page.tsx

```diff
@@ -1,47 +1,29 @@
 import { NextPage } from "next";
 import { useRouter } from "next/router";
 import Link from "next/link";
-import { event as gaEvent } from "react-ga";
```

Comment:
Question: does removing this break/affect our telemetry?

Reply:
Note: I double-checked and it exists (as it did before) in `PlanMatrix.tsx` so other components are not affected.